docs/react/api-angular/executors/delegate-build.md

```diff
@@ -0,0 +1,26 @@
+# delegate-build
+
+Delegates the build to a different target while supporting incremental builds
+
+Properties can be configured in workspace.json when defining the executor, or when invoking it.
+Read more about how to use executors and the CLI here: https://nx.dev/react/getting-started/nx-cli#running-tasks.
+
+## Properties
+
+### buildTarget
+
+Type: `string`
+
+Build target used for building the app after its dependencies have been built. If no target is configured, @angular-devkit/build-angular:browser is sheduled directly.
```

Comment:
I don't think we want a default here.

If the user hasn't defined a `buildTarget` we should throw a `SchematicsException`.

This builder is something that the developer needs to decide to use themselves, they don't get it out of the box, so their purpose for using it should be to get incremental build support for a different `target` they have defined in their `angular.json` or `workspace.json`. 

```json

"targets": {
  "build-app": {
    "builder": "ngx-build-plus",
     "outputPath": "dist/",
     ...
  },
  "build": {
     "builder": "@nrwl/angular:delegate-build",
     "buildTarget": "build-app",
     "tsConfig": "path/tsconfig.json"
   }
}

```

We don't want to mix concerns here that the `buildTarget` could be a `target` or a `builder`.

Reply:
No, we don't. It's a bad copy, the functionality is not doing that. I'll amend the docs.

Regarding throwing an exception, that's not needed, if `buildTarget` is not specified, that will be caught by the schema validation phase and an error will be thrown asking for the required options.